### PR TITLE
Add Alexbonjour bio

### PR DESCRIPTION
**Pull request by alexbonjour**
Adds a bio for @alexbonjour

What do you think @mpersiko?

### CODE REVIEW

**Comment by phongta** [comment]
test comment

**Comment by alexbonjour** [comment]
:+1:

File: bios/alexbonjour.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## alexbonjour
+
+**Name:** Alex
+
+**Occupation:** Github Student
```
Comment (alexbonjour) [inline bios/alexbonjour.md]: ??

**Comment by alexbonjour** [comment]
:+1: after fix.

**Pull request merged**